plugin/httptrace/clienttrace.go

```diff
@@ -56,6 +56,7 @@ func NewClientTrace(ctx context.Context) *httptrace.ClientTrace {
 
 	ct.tr = global.TraceProvider().Tracer("go.opentelemetry.io/otel/plugin/httptrace")
 	ct.start("http.request", "http.request")
+	defer ct.end("http.request", nil)
```

Comment:
Hey @huikang, thanks for digging on this issue.

The proposed solution ends the `http.request` span when the `NewClientTrace` call finishes, so this is actually just the struct initialization, not really the request. As you can see on the following screenshot:
![image](https://user-images.githubusercontent.com/7898464/71371761-0144e780-2591-11ea-9206-6dc676af4ba9.png)

I think the best solution for now is to remove the `http.request` span instead of ending it. Since there is a limitation of what we can trace with the `httptrace` package. And in the future refactor the http trace plugin to have a more robust way to trace a request.

Reply:
Thanks for your suggestion. So the http.request does being displayed on your UI screenshot. However, could you help to explain why the span does not show up using stdout exporter as in the example?

Reply:
This is a screenshot with the changes you submitted, that is reason it shows on the UI :)

Reply:
Gotcha. That makes sense in that the request span does not properly reflect the request is complete. I will update the PR based on your suggestion.

Reply:
Exactly! Thanks.